client/src/sass/application.scss

```diff
@@ -338,3 +361,32 @@ table {
     }
   }
 }
+
+@media screen and (min-width: $small-view) and  (max-width: #{$small-view + $menu-width}) {
+  .main-col {
+    &:not(.expanded) {
+      .admin-sub-header {
+        flex-direction: column;
+
+        .form-sub-title {
+          margin-right: 0px !important;
+          margin-bottom: 10px;
+          text-align: center;
+        }
+
+        .admin-sub-nav {
+          display: block;
+          overflow-x: auto;
+          white-space: nowrap;
+          height: 50px;
+          padding: 10px 0;
+          width: calc(100vw - #{$menu-width + ($expanded-horizontal-margins/3) *2});
```

Comment:
since this is the only varying part in the whole block and that you repeat twice, generally make a mixin to factorize it. Mixin are inexpensive but greatly improve maintainability (since then you spot immediately what the pivoting part between two usages is.

Reply:
I've placed the mixin into `sass/include/_mixins.scss`, right place for that kind of mixin ?

Reply:
Yes ; if a mixin is local to a component style file, we place it there, otherwise it goes in mixins. There are some exceptions for the miniature, but otherwise that's pretty much our mixin library. It only adds to a SCSS file weight if you specifically `@include` it, so you can put as many as you want there.

Reply:
thanks for the explanations and all the time you give to review all these PR :)

Reply:
You're welcome :slightly_smiling_face: Keep up the good work!